.dockerignore

```diff
@@ -1 +1 @@
-/bazel-cache
+/*
```

Comment:
Maybe add a comment saying that we don't need anything from the docker context when building the image.

Reply:
Done